### PR TITLE
changes to address git issue#216

### PR DESCRIPTION
**Pull request by manishjha86**
Signed-off-by: Manish Jha <er.manish.jha@gmail.com>

### CODE REVIEW

**Comment by laidbackware** [comment]
@manishjha86 Please remove the changes to the Harbor section, as this is being addresses with https://github.com/vmware-tanzu-labs/tanzu-validated-solutions/pull/226 and addressing again here will likely cause more issues.

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 1]
```diff
@@ -129,19 +130,14 @@ The CNI options are open-source software that provide networking for cluster pod
...
-- [Deploy Tanzu Kubernetes clusters with Calico](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.5/vmware-tanzu-kubernetes-grid-15/GUID-tanzu-k8s-clusters-networking.html#calico)
-
-- [Implement Multiple Pod Network Interfaces with Multus](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.5/vmware-tanzu-kubernetes-grid-15/GUID-packages-cni-multus.html)
+To provision a Tanzu Kubernetes cluster using Calico CNI, see [Deploy Tanzu Kubernetes clusters with Calico](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.5/vmware-tanzu-kubernetes-grid-15/GUID-tanzu-k8s-clusters-networking.html#calico)
 
 Each CNI is suitable for a different use case. The following table lists some common use cases for the CNI options that Tanzu Kubernetes Grid supports. This table will help you select the most appropriate CNI for your Tanzu Kubernetes Grid implementation.
 
 |**CNI**|**Use Case**|**Pros and Cons**|
-| --- | --- | --- | --- |
+| :- | :- | :- |
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:138]: replace :- with ---

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 2]
```diff
@@ -334,9 +332,10 @@ For more information about Contour, see the [Contour](https://projectcontour.io/
 Each ingress controller has pros and cons of its own. The below table provides general recommendations on when you should use a specific ingress controller for your Kubernetes environment.
 
 |**Ingress Controller**|**Use Cases**|
-| --- | --- | --- | --- |
+| :- | :- |
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:335]: replace :- with ---

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 3]
```diff
@@ -334,9 +332,10 @@ For more information about Contour, see the [Contour](https://projectcontour.io/
...
 |Contour|<p>Use Contour when only north-south traffic is needed in a Kubernetes cluster. You can apply security policies for the north-south traffic by defining the policies in the manifest file for the application.</p><p></p><p>Contour is a reliable solution for simple Kubernetes workloads. </p>|
 |Istio|Use Istio ingress controller when you need to provide security, traffic direction, and insight within the cluster (east-west traffic) and between the cluster and the outside world (north-south traffic).|
+|NSX Advanced Load Balancer Ingress controller|Use NSX Advanced Load Balancer ingress controller when a containerized application requires features like local and global server load balancing (GSLB), web application firewall (WAF), performance monitoring, etc.|
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:338]: ... when a containerized application requires features such as local and global server ... (WAF), and performance monitoring.|

**Review by srinivasme21 — CHANGES_REQUESTED** [review]
Please fix IX review comments.

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 4]
```diff
@@ -377,8 +376,6 @@ When vSphere with Tanzu is deployed on VDS networking, you can deploy an externa
 
 There are two main supported installation methods for Harbor:
```
[the remark this answers is not in the document]
Comment (manishjha86) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md:377]: I have replaced the statement as "The supported installation method for Harbor in a vSphere with Tanzu environment is described below"

File: src/reference-designs/tko-on-vsphere-with-tanzu.md (hunk 3 above)
Comment (manishjha86) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md]: I have removed the nsx alb ingress controller as vsphere with tanzu doesn't support ALB L7 ingress

**Review by srinivasme21 — APPROVED** [review]
LGTM

**Pull request merged**